src/test/java/io/dropwizard/sharding/dao/locktest/LockTest.java

```diff
@@ -113,4 +114,75 @@ public void testLockingFail() throws Exception {
                 .execute();
 
     }
+
+
+
+    @Test
+    public void testPersist() throws Exception {
+        SomeLookupObject p1 = SomeLookupObject.builder()
+                .myId("0")
+                .name("Parent 1")
+                .build();
+
+        lookupDao.lockedExecutor(p1)
```

Comment:
Let's create two methods:
1) One that saves and returns a locked context: saveAndGetExecutor
2) One that locks and existing object: lockAndGetExecutor